# 365 Probabilidades · Dia #077
## Qual a probabilidade de a sua mala não chegar com você?

**Tipo:** Descritivo
**Data de publicação:** 2026-08-29
**Ferramenta:** Python
**Decisão analisada:** Devo despachar a mala?
**Hashtag:** #365Probabilidades #Dia077

---

### 📖 A História

Existe um momento específico, na esteira de bagagem, em que a pessoa começa a
calcular.

As malas param de sair. Ainda tem gente esperando, mas menos. E aí começa a conta
silenciosa: quantas voltas a esteira já deu, quantas pessoas ainda estão aqui, será
que a minha vem.

É uma das poucas situações da vida moderna em que a gente estima probabilidade de
forma explícita, olhando para uma esteira vazia.

E é uma conta que dá para fazer de verdade, porque o número existe e é publicado
todo ano.

---

### 📚 O Conceito: uma exceção declarada de fonte

Aqui eu preciso abrir uma exceção ao critério do projeto, e prefiro fazer isso na
cara do leitor.

Todos os dias desta série usam fonte primária revisada por pares. **Este não.**

O dado de bagagem vem do relatório anual da SITA, que é uma fornecedora de tecnologia
de rastreamento de bagagem para companhias aéreas e aeroportos. É um relatório de
indústria, com metodologia proprietária, cobertura que varia entre companhias, e um
interesse comercial embutido: quanto mais visível o problema, melhor para quem vende
a solução.

Não é motivo para descartar. É motivo para ler com outro tipo de atenção, e existe uma
prova interna de que essa atenção é necessária.

Por anos, a indústria inteira repetiu que uma mala mal manuseada custava **US$ 150**.
No relatório de 2026, a própria SITA substituiu essa cifra por **US$ 260**, dizendo
que agora o número vem de um modelo transparente e baseado em dados. Ou seja: o valor
anterior circulava sem modelo público, e ninguém tinha conferido.

Um dia sobre número de bagagem vira, sem querer, um dia sobre de onde vêm os números
que a gente repete.

---

### 🧮 O Modelo

Contagem administrativa da indústria aérea, com composição por tipo de ocorrência.

**Fontes:**
- SITA · *Baggage IT Insights 2025*, com dados de 2024 · **taxa global de 6,3 malas
  mal manuseadas por 1.000 passageiros**, contra 6,9 em 2023 e 7,6 em 2022 ·
  **67% de melhora desde 2007**
- **Total absoluto: 36,2 milhões** de malas mal manuseadas em 2024, contra
  **36,1 milhões** em 2023
- Tráfego global de **5,3 bilhões de passageiros**, com crescimento de **8,2%** em 2024
- **Composição:** atrasadas **74%** (contra 80% no ano anterior), perdidas ou roubadas
  **8%**, danificadas ou saqueadas **18%** (contra 15%)
- **Origem:** transferência entre voos é o maior contribuinte, com **41%** dos casos,
  contra 46% no ano anterior
- **Melhor região:** Ásia-Pacífico, com **3,1 por 1.000**, praticamente estável desde
  2007
- SITA · *Baggage IT Insights 2026* · custo estimado de **US$ 260 por mala mal
  manuseada**, substituindo a cifra de **US$ 150** amplamente citada até então · custo
  total de **US$ 6,3 bilhões**

**Exceção de fonte declarada:** relatório de indústria, não artigo revisado por pares.
O fornecedor do dado vende sistemas de rastreamento de bagagem.

**Nota metodológica sobre o fator ×0.80:** não se aplica. Contagem administrativa de
bagagens processadas, não proporção de survey autorrelatado.


In [1]:
import numpy as np
import matplotlib.pyplot as plt

plt.rcParams['font.family'] = 'serif'
plt.rcParams['axes.spines.top'] = False
plt.rcParams['axes.spines.right'] = False
plt.rcParams['axes.grid'] = True
plt.rcParams['grid.alpha'] = 0.3
plt.rcParams['figure.facecolor'] = 'white'
plt.rcParams['axes.facecolor'] = 'white'

print("Bibliotecas carregadas")

Bibliotecas carregadas


In [2]:
# --- DADOS DA LITERATURA ---
# SITA, Baggage IT Insights 2025 (dados de 2024) e 2026

# Taxa por mil passageiros
TAXA = {2007: None, 2022: 7.6, 2023: 6.9, 2024: 6.3}
melhora_desde_2007 = 0.67

# Contagem absoluta de malas mal manuseadas
TOTAL_MALAS = {2023: 36_100_000, 2024: 36_200_000}

# Trafego
passageiros_2024 = 5_300_000_000
crescimento_2024 = 0.082

# Composicao das malas mal manuseadas, 2024 (com o ano anterior entre parenteses)
COMPOSICAO = {
    'Atrasadas':               (0.74, 0.80),
    'Danificadas ou saqueadas': (0.18, 0.15),
    'Perdidas ou roubadas':    (0.08, None),
}

# Origem do problema
p_transferencia = 0.41      # contra 0,46 no ano anterior

# Melhor regiao
taxa_asia_pacifico = 3.1

# Custo por mala
custo_atual = 260           # SITA 2026, modelo declarado
custo_antigo = 150          # cifra amplamente citada antes, sem modelo publico
custo_total_bi = 6.3

aplica_fator_080 = False

print("=" * 70)
print("  DADOS - BAGAGEM MAL MANUSEADA")
print("=" * 70)
print(f"\n  Taxa global por 1.000 passageiros:")
for ano, t in TAXA.items():
    if t is not None:
        print(f"  -> {ano}: {t}")
print(f"  -> Melhora desde 2007: {melhora_desde_2007*100:.0f}%")
print(f"  -> Melhor regiao (Asia-Pacifico): {taxa_asia_pacifico}")
print(f"\n  Contagem absoluta:")
for ano, n in TOTAL_MALAS.items():
    print(f"  -> {ano}: {n:,}".replace(",", ".") + " malas")
print(f"\n  Trafego 2024: {passageiros_2024:,}".replace(",", ".") +
      f" passageiros (+{crescimento_2024*100:.1f}%)")
print(f"\n  Composicao das malas mal manuseadas em 2024:")
for tipo, (atual, anterior) in COMPOSICAO.items():
    extra = f"  (era {anterior*100:.0f}%)" if anterior else ""
    print(f"  -> {tipo:<26} {atual*100:>4.0f}%{extra}")
print(f"\n  Maior origem: transferencia entre voos, {p_transferencia*100:.0f}% dos casos")
print(f"\n  Custo por mala mal manuseada:")
print(f"  -> Cifra antiga, amplamente citada:  US$ {custo_antigo}")
print(f"  -> Modelo declarado da SITA em 2026: US$ {custo_atual}")
print(f"  -> Custo total: US$ {custo_total_bi} bilhoes")
print(f"\n  Fator x0.80 aplicado: {aplica_fator_080}")
print("=" * 70)

  DADOS - BAGAGEM MAL MANUSEADA

  Taxa global por 1.000 passageiros:
  -> 2022: 7.6
  -> 2023: 6.9
  -> 2024: 6.3
  -> Melhora desde 2007: 67%
  -> Melhor regiao (Asia-Pacifico): 3.1

  Contagem absoluta:
  -> 2023: 36.100.000 malas
  -> 2024: 36.200.000 malas

  Trafego 2024: 5.300.000.000 passageiros (+8.2%)

  Composicao das malas mal manuseadas em 2024:
  -> Atrasadas                    74%  (era 80%)
  -> Danificadas ou saqueadas     18%  (era 15%)
  -> Perdidas ou roubadas          8%

  Maior origem: transferencia entre voos, 41% dos casos

  Custo por mala mal manuseada:
  -> Cifra antiga, amplamente citada:  US$ 150
  -> Modelo declarado da SITA em 2026: US$ 260
  -> Custo total: US$ 6.3 bilhoes

  Fator x0.80 aplicado: False


In [3]:
# --- O MODELO ---

# 1) O PARADOXO: a taxa cai e a contagem sobe
var_taxa = (TAXA[2024] - TAXA[2023]) / TAXA[2023]
var_total = (TOTAL_MALAS[2024] - TOTAL_MALAS[2023]) / TOTAL_MALAS[2023]

# 2) O que "mal manuseada" realmente significa
p_mal_manuseada = TAXA[2024] / 1000
p_perdida = p_mal_manuseada * COMPOSICAO['Perdidas ou roubadas'][0]
p_atrasada = p_mal_manuseada * COMPOSICAO['Atrasadas'][0]
p_danificada = p_mal_manuseada * COMPOSICAO['Danificadas ou saqueadas'][0]
um_em_quantos = 1 / p_perdida
um_em_quantos_geral = 1 / p_mal_manuseada

# 3) Risco acumulado ao longo de varias viagens
#    PREMISSA DECLARADA: taxa por passageiro, viagens independentes.
#    Nao e exato: a taxa e por passageiro processado, e conexao muda o risco.
def acumulado(p, n):
    return 1 - (1 - p) ** n


viagens = [1, 2, 4, 10, 20, 50]

# 4) Traducao do custo
custo_total_calc = TOTAL_MALAS[2024] * custo_atual / 1e9
diferenca_cifras = (custo_atual - custo_antigo) / custo_antigo

print("=" * 70)
print("  MODELO - O MESMO ANO, DUAS MANCHETES OPOSTAS")
print("=" * 70)
print(f"\n  1) O paradoxo do numerador e do denominador:")
print(f"  -> A TAXA caiu {abs(var_taxa)*100:.1f}%"
      f" (de {TAXA[2023]} para {TAXA[2024]} por mil)")
print(f"  -> A CONTAGEM subiu {var_total*100:.1f}%"
      f" (de {TOTAL_MALAS[2023]:,}".replace(",", ".") +
      f" para {TOTAL_MALAS[2024]:,}".replace(",", ".") + ")")
print(f"  -> Duas manchetes verdadeiras no mesmo ano:")
print(f"     'melhor taxa da historia' e 'mais malas extraviadas que no ano passado'")
print(f"  -> O denominador cresceu {crescimento_2024*100:.1f}%. Foi so isso.")
print(f"\n  2) O que 'mal manuseada' realmente quer dizer:")
print(f"  -> Chance de a mala ter QUALQUER problema:"
      f" {p_mal_manuseada*100:.2f}%  (1 em {um_em_quantos_geral:.0f})")
print(f"     -> atrasada e depois entregue: {p_atrasada*1000:.2f} por mil")
print(f"     -> danificada ou saqueada:     {p_danificada*1000:.2f} por mil")
print(f"     -> PERDIDA ou roubada:         {p_perdida*1000:.2f} por mil")
print(f"  -> Chance de perder a mala DE VERDADE:"
      f" 1 em {um_em_quantos:.0f}")
print(f"  -> O numero que assusta e o que importa diferem por um fator de"
      f" {p_mal_manuseada/p_perdida:.0f}.")
print(f"\n  3) Risco acumulado (PREMISSA DECLARADA: viagens independentes):")
print(f"  {'viagens':>8} | {'algum problema':>15} | {'perda de verdade':>17}")
print(f"  {'-'*8}-+-{'-'*15}-+-{'-'*17}")
for n in viagens:
    print(f"  {n:>8} | {acumulado(p_mal_manuseada, n)*100:>14.2f}%"
          f" | {acumulado(p_perdida, n)*100:>16.2f}%")
print(f"\n  4) Onde o problema nasce:")
print(f"  -> {p_transferencia*100:.0f}% dos casos acontecem em TRANSFERENCIA entre voos")
print(f"  -> Voo direto e a unica variavel que o passageiro controla de fato")
print(f"\n  5) O custo, e a licao sobre numeros repetidos:")
print(f"  -> {TOTAL_MALAS[2024]:,}".replace(",", ".") +
      f" malas x US$ {custo_atual} = US$ {custo_total_calc:.1f} bilhoes")
print(f"  -> A cifra anterior, US$ {custo_antigo}, circulou por anos sem modelo publico")
print(f"  -> A correcao foi de +{diferenca_cifras*100:.0f}%")
print(f"\n  ATENCAO, INCONSISTENCIA NA PROPRIA FONTE:")
print(f"  -> A conta acima da US$ {custo_total_calc:.1f} bilhoes.")
print(f"  -> A SITA publica US$ {custo_total_bi} bilhoes de custo total.")
print(f"  -> Diferenca de US$ {custo_total_calc - custo_total_bi:.1f} bilhoes.")
print(f"  -> As duas cifras vem da mesma fonte e nao fecham entre si.")
print(f"     Podem se referir a anos, bases ou escopos diferentes.")
print(f"     Registrado, nao resolvido. Este notebook nao escolhe uma delas.")
print("=" * 70)

  MODELO - O MESMO ANO, DUAS MANCHETES OPOSTAS

  1) O paradoxo do numerador e do denominador:
  -> A TAXA caiu 8.7% (de 6.9 para 6.3 por mil)
  -> A CONTAGEM subiu 0.3% (de 36.100.000 para 36.200.000)
  -> Duas manchetes verdadeiras no mesmo ano:
     'melhor taxa da historia' e 'mais malas extraviadas que no ano passado'
  -> O denominador cresceu 8.2%. Foi so isso.

  2) O que 'mal manuseada' realmente quer dizer:
  -> Chance de a mala ter QUALQUER problema: 0.63%  (1 em 159)
     -> atrasada e depois entregue: 4.66 por mil
     -> danificada ou saqueada:     1.13 por mil
     -> PERDIDA ou roubada:         0.50 por mil
  -> Chance de perder a mala DE VERDADE: 1 em 1984
  -> O numero que assusta e o que importa diferem por um fator de 12.

  3) Risco acumulado (PREMISSA DECLARADA: viagens independentes):
   viagens |  algum problema |  perda de verdade
  ---------+-----------------+------------------
         1 |           0.63% |             0.05%
         2 |           1.26% |    

In [4]:
# --- VISUALIZACAO ---

DOURADO = '#c8a84b'
VERMELHO = '#c0392b'
VERDE = '#2a8a82'
CINZA = '#6b6a64'

# GRAFICO 1 - Assinatura: a taxa cai e a contagem sobe
fig1, (ax1a, ax1b) = plt.subplots(1, 2, figsize=(14, 7))

anos = [2022, 2023, 2024]
taxas = [TAXA[a] for a in anos]
ax1a.plot(anos, taxas, marker='o', markersize=15, linewidth=3.2, color=VERDE)
for a, t in zip(anos, taxas):
    ax1a.text(a, t + 0.09, f'{t}'.replace('.', ','), ha='center', fontsize=19,
              fontweight='bold', color=VERDE)
ax1a.set_xticks(anos)
ax1a.set_ylim(5.9, 8.1)
ax1a.set_ylabel('Malas mal manuseadas por 1.000 passageiros')
ax1a.set_title('A TAXA cai\nmelhor marca desde que se mede', fontsize=15, pad=14)

anos2 = [2023, 2024]
totais = [TOTAL_MALAS[a] / 1e6 for a in anos2]
barras = ax1b.bar([str(a) for a in anos2], totais, color=[CINZA, VERMELHO],
                  alpha=0.9, width=0.45)
for barra, v in zip(barras, totais):
    ax1b.text(barra.get_x() + barra.get_width() / 2, v + 0.25,
              f'{v:.1f} mi'.replace('.', ','), ha='center', fontsize=19,
              fontweight='bold')
ax1b.set_ylim(0, 42)
ax1b.set_ylabel('Total de malas mal manuseadas (milhões)')
ax1b.set_title('A CONTAGEM sobe\nmais malas que no ano anterior', fontsize=15, pad=14)

fig1.suptitle('O mesmo ano, duas manchetes verdadeiras e opostas\n'
              'O denominador cresceu 8,2%. Foi só isso.',
              fontsize=16, y=1.04)
plt.figtext(0.5, 0.005,
            'Fonte: SITA, Baggage IT Insights 2025 (dados de 2024)  |  '
            '#365Probabilidades',
            ha='center', fontsize=9, color='gray')
plt.tight_layout()
plt.savefig('dia-077-grafico-01-taxa-e-contagem.png', dpi=150, bbox_inches='tight')
plt.close()
print("Grafico 1 salvo")

# GRAFICO 2 - O que "mal manuseada" quer dizer
fig2, ax2 = plt.subplots(figsize=(12, 7))

tipos = list(COMPOSICAO.keys())
valores = [COMPOSICAO[t][0] * 100 for t in tipos]
cores = [DOURADO, CINZA, VERMELHO]

esquerda = 0
for tipo, valor, cor in zip(tipos, valores, cores):
    ax2.barh([0], [valor], left=esquerda, color=cor, alpha=0.9, height=0.42)
    ax2.text(esquerda + valor / 2, 0, f'{valor:.0f}%', ha='center', va='center',
             fontsize=21, fontweight='bold', color='white')
    dy = -0.33 if tipo == 'Atrasadas' else (-0.33 if tipo == 'Danificadas ou saqueadas'
                                            else -0.60)
    ax2.text(esquerda + valor / 2, dy, tipo, ha='center', va='top', fontsize=13,
             color=cor)
    esquerda += valor

ax2.set_xlim(0, 100)
ax2.set_ylim(-1.1, 0.75)
ax2.set_yticks([])
ax2.grid(False)
ax2.set_xlabel('Composição das malas mal manuseadas (%)')
ax2.set_title('"Mal manuseada" não é "perdida"\n'
              'Três em cada quatro apenas atrasam e chegam depois',
              fontsize=15, pad=18)
ax2.text(50, 0.55,
         f'Chance de perder a mala de verdade: 1 em {um_em_quantos:,.0f}'
         .replace(",", "."),
         ha='center', fontsize=15, fontweight='bold', color=VERMELHO)

plt.figtext(0.5, 0.005,
            'Fonte: SITA, Baggage IT Insights 2025  |  #365Probabilidades',
            ha='center', fontsize=9, color='gray')
plt.tight_layout()
plt.savefig('dia-077-grafico-02-composicao.png', dpi=150, bbox_inches='tight')
plt.close()
print("Grafico 2 salvo")

# GRAFICO 3 - Risco acumulado ao longo das viagens
fig3, ax3 = plt.subplots(figsize=(12, 8))

ns = np.arange(1, 51)
ax3.plot(ns, [acumulado(p_mal_manuseada, n) * 100 for n in ns], color=DOURADO,
         linewidth=3.2, label='algum problema com a mala')
ax3.plot(ns, [acumulado(p_perdida, n) * 100 for n in ns], color=VERMELHO,
         linewidth=3.2, label='perda de verdade')

for n in (10, 50):
    ax3.scatter([n], [acumulado(p_mal_manuseada, n) * 100], s=140, color=DOURADO,
                zorder=3)
    ax3.text(n, acumulado(p_mal_manuseada, n) * 100 + 0.9,
             f'{acumulado(p_mal_manuseada, n)*100:.1f}%'.replace('.', ','),
             ha='center', fontsize=13, fontweight='bold', color=DOURADO)

ax3.set_xlim(0, 51)
ax3.set_ylim(0, 32)
ax3.set_xlabel('Número de viagens')
ax3.set_ylabel('Chance acumulada de acontecer ao menos uma vez (%)')
ax3.legend(frameon=False, fontsize=13, loc='upper left')
ax3.set_title('Ao longo de uma vida de viagens\n'
              'Cinquenta voos e a chance de algum problema ainda não chega a um terço',
              fontsize=15, pad=18)
ax3.text(0.5, -0.135,
         'Premissa declarada: taxa aplicada por viagem e viagens independentes. '
         'A taxa publicada é por passageiro processado,\ne 41% dos casos nascem em '
         'transferência, então quem faz conexão tem risco maior que esta curva.',
         transform=ax3.transAxes, ha='center', fontsize=10, color=CINZA, style='italic')

plt.figtext(0.5, 0.005,
            'Cálculo próprio sobre a taxa publicada pela SITA  |  #365Probabilidades',
            ha='center', fontsize=9, color='gray')
plt.tight_layout()
plt.savefig('dia-077-grafico-03-acumulado.png', dpi=150, bbox_inches='tight')
plt.close()
print("Grafico 3 salvo")

Grafico 1 salvo
Grafico 2 salvo
Grafico 3 salvo


### 💡 O Insight

**6,3 malas a cada mil passageiros.**

Ou seja, cerca de **0,63%**, uma em cada 159 pessoas. É a melhor taxa desde que a
indústria mede, 67% melhor que em 2007.

E, no mesmo ano, o **total** de malas mal manuseadas **subiu**: de 36,1 milhões para
36,2 milhões.

As duas frases são verdadeiras. A taxa caiu e a contagem aumentou, porque o
denominador cresceu 8,2%. Dá para escrever "melhor ano da história do setor" e "mais
malas extraviadas que no ano passado" com os mesmos dados, na mesma tabela.

Este projeto já teve um dia inteiro sobre isso, o #066. A diferença é que lá era uma
universidade em 1973 e aqui é a sua mala.

O segundo número muda mais ainda a leitura.

"Mal manuseada" não quer dizer perdida. **Três em cada quatro casos são apenas atraso**,
mala que chega no voo seguinte. Perda ou roubo são só 8% do total.

Fazendo a conta: a chance de a sua mala ter algum problema é de 1 em 159. A chance de
**perder a mala de verdade** é de cerca de **1 em 1.984**.

O número que assusta e o número que importa diferem por um fator de doze.

E tem a variável que você controla: **41% dos casos nascem em transferência entre
voos.** Não em despachar, não em aeroporto ruim. Na conexão.

Agora, a parte que me interessou mais e que não estava no plano deste dia.

A fonte disso tudo é um relatório de indústria, produzido por uma empresa que **vende
sistemas de rastreamento de bagagem**. Não é artigo revisado por pares, e é a primeira
vez que este projeto usa uma fonte assim. Estou dizendo isso porque acho que precisa
ser dito, e não em nota de rodapé.

E aí apareceu a melhor evidência de que a desconfiança vale a pena.

Por anos, todo texto sobre bagagem repetiu que uma mala mal manuseada custa **US$ 150**
à companhia aérea. No relatório de 2026, a própria SITA trocou essa cifra por
**US$ 260**, explicando que agora existe um modelo transparente e baseado em dados
atrás do número.

Traduzindo: o valor que circulou pelo mundo inteiro por anos não tinha modelo público.
Ele era repetido porque era repetido.

Então o dia da mala virou, sem querer, um dia sobre a origem dos números.

*Qual número você repete com segurança sem nunca ter procurado a fonte?*

---

### ⚠️ Limitações do Modelo

- **Exceção de fonte declarada.** Este é o primeiro dia do projeto baseado em relatório
  de indústria, e não em artigo revisado por pares. O fornecedor do dado, a SITA, vende
  sistemas de rastreamento de bagagem para o setor que ela mede. Metodologia
  proprietária e cobertura variável entre companhias.
- **A taxa é por passageiro processado, não por mala nem por trecho.** Quem despacha
  duas malas, ou faz várias conexões, não está descrito exatamente por esse número.
- **A curva de risco acumulado é cálculo próprio**, com premissa declarada de viagens
  independentes e taxa aplicada por viagem. Como 41% dos casos nascem em transferência,
  quem faz conexão tem risco maior que a curva mostra, e quem só voa direto tem menos.
- **Média global esconde variação enorme.** A Ásia-Pacífico tem 3,1 por mil, metade da
  média mundial. Não existe uma taxa que descreva o seu aeroporto.
- **"Mal manuseada" é categoria da indústria**, não do passageiro. Uma mala que chega
  três dias depois e uma que some para sempre entram na mesma contagem, ainda que a
  composição por tipo esteja publicada.
- **A cifra de custo mudou de US$ 150 para US$ 260 pela própria fonte**, o que é uma
  boa razão para tratar qualquer número desse relatório como estimativa em revisão, e
  não como medida fechada.
- **As cifras de custo da própria fonte não fecham entre si.** 36,2 milhões de malas
  multiplicadas por US$ 260 dão cerca de US$ 9,4 bilhões, mas a SITA publica um custo
  total de US$ 6,3 bilhões. A diferença é de mais de três bilhões de dólares. As duas
  cifras podem se referir a anos, bases ou escopos diferentes, e o relatório acessível
  não permite decidir. **Registrado e não resolvido:** este notebook não escolhe uma
  das duas, e nenhuma delas deve ser citada como fato fechado.
- Fator ×0.80 não aplicado: contagem administrativa, não survey.

*A ciência é honesta sobre o que não sabe. O modelo também.*

---

### 📎 Links
- Substack: [link do post]
- Instagram: [link do post]

---
*365 Probabilidades · Decidindo com dados, um dia de cada vez.*
